### PR TITLE
Adding support to configure execution mode

### PR DESCRIPTION
**Pull request by sudomateo**
Attempting to add the ability to manage execution mode via the SDK. Looking for some feedback and also help with adding this to tests.

Requires https://github.com/hashicorp/go-tfe/pull/84 to be merged first.

### CODE REVIEW

File: tfe/data_source_workspace.go [hunk 1]
```diff
@@ -86,6 +86,11 @@ func dataSourceTFEWorkspace() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"remote_execution": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
```
Comment (svanharmelen) [inline tfe/data_source_workspace.go]: I tried to keep these parameters somewhat (not fully) alphabetically ordered. So in the spirit, this new parameter should probably be placed between `queue_all_runs` and `ssh_key_id`.

File: tfe/data_source_workspace.go [hunk 2]
```diff
@@ -114,6 +119,7 @@ func dataSourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("trigger_prefixes", workspace.TriggerPrefixes)
 	d.Set("working_directory", workspace.WorkingDirectory)
 	d.Set("external_id", workspace.ID)
+	d.Set("remote_execution", workspace.Operations)
```
Comment (svanharmelen) [inline tfe/data_source_workspace.go]: Same here about the ordering :wink:

File: tfe/resource_tfe_workspace.go [hunk 3]
```diff
@@ -108,6 +108,12 @@ func resourceTFEWorkspace() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"remote_execution": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
```
Comment (svanharmelen) [inline tfe/resource_tfe_workspace.go]: I tried to keep these parameters somewhat (not fully) alphabetically ordered. So in the spirit, this new parameter should probably be placed between `queue_all_runs` and `terraform_version`.

File: tfe/resource_tfe_workspace.go [hunk 4]
```diff
@@ -248,6 +255,7 @@ func resourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("trigger_prefixes", workspace.TriggerPrefixes)
 	d.Set("working_directory", workspace.WorkingDirectory)
 	d.Set("external_id", workspace.ID)
+	d.Set("remote_execution", workspace.Operations)
```
Comment (svanharmelen) [inline tfe/resource_tfe_workspace.go]: Same here about the ordering 😉

**Comment by sudomateo** [comment]
Thank you again for the feedback. I pushed some changes to implement the changes. I'm okay with renaming this `operations` to stay consistent with the rest of the resource.

**Comment by svanharmelen** [comment]
Fixes #87

**Pull request merged**